mlir/lib/Dialect/Rock/Tuning/RockTuningImpl.cpp

```diff
@@ -643,6 +644,14 @@ LogicalResult getTuningProblemStr(rock::RockGemmWrapperInterface gemmIF,
   KernelType opType = gemmIF.getKernelType();
   Operation *gemmOp = gemmIF.getOperation();
 
+  auto f8TypeStr = [](const Type &type) -> const char * {
```

Comment:
Abstraction, for forming the op field of the tuning key.

Reply:
I don't like this.

I'd do a `std::optional<StringLiteral> toShortFp8Name()` and then `llvm::formatv("conv{0}_{1}", ...)` to form the name.

If nothing else, `const char*` doesn't get used much in this codebase

Reply:
formatv() is what I needed.  Formatted output is better than << for anything half-complicated.